### PR TITLE
feat: add zeplin integration to automate process

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

File: tokens/zeplin.json [hunk 1]
```diff
@@ -0,0 +1,3444 @@
...
+					"size": 12,
+					"weight": 400,
+					"stretch": 1
+				}
+			},
+			"woff": { "value": "assets/fonts/dbscreensans-regular.woff" },
+			"woff2": { "value": "assets/fonts/dbscreensans-regular.woff2" }
+		}
+	}
+}
```
Comment (mfranzke) [inline tokens/zeplin.json:3444]: I think that this even already shows the downside of this procedure: We wouldn't need to include all of those tokens, as only a handful is really necessary.

File: tokens/font.jsonc [hunk 2]
```diff
@@ -3,14 +3,14 @@
 		"family": {
 			// These first two ones are the only two that are necessary tokens, the three at the end are only relevant for tailwind
 			"base": {
-				"value": "'{asset.font.db-screensans-regular.fontfamily.value}', Helvetica, Arial, sans-serif"
+				"value": "'DBScreenSans', Helvetica, Arial, sans-serif"
 			},
 			"headline": {
-				"value": "'{asset.font.db-screenhead.fontfamily.value}', Helvetica, Arial, sans-serif"
+				"value": "'DBScreenHead', Helvetica, Arial, sans-serif"
```
Comment (mfranzke) [inline tokens/font.jsonc:9]: These values were actually correct previously and are now incorrect.

File: tokens/font.jsonc [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
 			},
-			"sans": "'{asset.font.db-screensans-regular.name.value}', Helvetica, Arial, sans-serif",
-			"bold": "'{asset.font.db-screensans-bold.name.value}', Helvetica, Arial, sans-serif",
-			"head": "'{asset.font.db-screenhead.name.value}', Helvetica, Arial, sans-serif"
+			"sans": "'DBScreenSans', Helvetica, Arial, sans-serif",
+			"bold": "'DBScreenSans', Helvetica, Arial, sans-serif",
+			"head": "'DBScreenHead', Helvetica, Arial, sans-serif"
```
Comment (mfranzke) [inline tokens/font.jsonc:13]: Why couldn't we reference those values anymore?

File: .env.template [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# It will force the user to add an e-mail for this project, before committing.
+COMMIT_MAIL=my.commit@mail.com
+
+# (https://app.zeplin.io/profile/developer)
+DEVELOPER_ZEPLIN_ACCESS_TOKEN=TODO
```
Comment (mfranzke) [inline .env.template:5]: What does this have to do with this topic / pull request ?

Comment (nmerget) [reply .env.template:5]: https://github.com/db-ui/base/blob/feat-zeplin-token-generation/scripts/zeplin-styleguide.js#L7

We need this as an access token to fetch the json from zeplin

File: tokens/zeplin.json (hunk 1 above)
Comment (nmerget) [reply tokens/zeplin.json:3444]: https://github.com/db-ui/base/blob/feat-zeplin-token-generation/scripts/zeplin-styleguide.js#L40

I add a TODO already. You are absolutely right, we need to make it more generic and merge all those. I guess it will only work if we make a commitment with UX on how we format everything. 
As for now the keys are separated by a hyphen so it is pretty easy fo us to merge them.

File: tokens/font.jsonc (hunk 2 above)
Comment (nmerget) [reply tokens/font.jsonc:9]: For now, I changed it to test the build process. We should ask UX here as well how we manage this.

**Comment by mfranzke** [comment]
On the console the following output warnings are shown – could we ignore this for the moment ?
```
While building tailwind-tokens.json, token collisions were found; output may be unexpected.
    Output name SpacingSpacing15 was generated by:
        spacing.spacing-1.5   12px
        spacing.spacing-15   120px
This many-to-one issue is usually caused by some combination of:
    * conflicting or similar paths/names in property definitions
    * platform transforms/transformGroups affecting names, especially when removing specificity
    * overly inclusive file filters
```

**Comment by nmerget** [comment]
> On the console the following output warnings are shown – could we ignore this for the moment ?
> 
> ```
> While building tailwind-tokens.json, token collisions were found; output may be unexpected.
>     Output name SpacingSpacing15 was generated by:
>         spacing.spacing-1.5   12px
>         spacing.spacing-15   120px
> This many-to-one issue is usually caused by some combination of:
>     * conflicting or similar paths/names in property definitions
>     * platform transforms/transformGroups affecting names, especially when removing specificity
>     * overly inclusive file filters
> ```

I saw this as well, and indeed this is an issue for flutter, ios, and js.

The only option would be to use something else than a point like "0.5" -> "0p5"

**Comment by mfranzke** [comment]
> > On the console the following output warnings are shown – could we ignore this for the moment ?
> > ```
> > While building tailwind-tokens.json, token collisions were found; output may be unexpected.
> >     Output name SpacingSpacing15 was generated by:
> >         spacing.spacing-1.5   12px
> >         spacing.spacing-15   120px
> > This many-to-one issue is usually caused by some combination of:
> >     * conflicting or similar paths/names in property definitions
> >     * platform transforms/transformGroups affecting names, especially when removing specificity
> >     * overly inclusive file filters
> > ```
> 
> I saw this as well, and indeed this is an issue for flutter, ios, and js.
> 
> The only option would be to use something else than a point like "0.5" -> "0p5"

What about some other characters like e.g. an underscore (or are those reserved / wouldn't follow that technologies code-styleguides)?

**Comment by nmerget** [comment]
> > > On the console the following output warnings are shown – could we ignore this for the moment ?
> > > ```
> > > While building tailwind-tokens.json, token collisions were found; output may be unexpected.
> > >     Output name SpacingSpacing15 was generated by:
> > >         spacing.spacing-1.5   12px
> > >         spacing.spacing-15   120px
> > > This many-to-one issue is usually caused by some combination of:
> > >     * conflicting or similar paths/names in property definitions
> > >     * platform transforms/transformGroups affecting names, especially when removing specificity
> > >     * overly inclusive file filters
> > > ```
> > 
> > 
> > I saw this as well, and indeed this is an issue for flutter, ios, and js.
> > The only option would be to use something else than a point like "0.5" -> "0p5"
> 
> What about some other characters like e.g. an underscore (or are those reserved / wouldn't follow that technologies code-styleguides)? "0.5" -> "0_5" (I know, not the best idea as the underscore is even also partly used as a replacement for a space, probably there are better ones)

Fixed it with the 0p5 solution, because 0_5 is not working :(

File: tokens/asset/font.json [hunk 5]
```diff
@@ -1,27 +0,0 @@
...
-			"db-screenhead": {
-				"name": { "value": "DB Screen Head" },
-				"fontfamily": { "value": "DB Screen Head" },
-				"fontweight": { "value": "300" },
-				"woff": { "value": "assets/fonts/dbscreenhead-light.woff" },
-				"woff2": { "value": "assets/fonts/dbscreenhead-light.woff2" }
-			}
-		}
-	}
-}
```
Comment (mfranzke) [inline tokens/asset/font.json:27]: Were do those tokens come from after your change? They wouldn't get delivered from Zeplin.

Comment (nmerget) [reply tokens/asset/font.json:27]: reverted it

**Review by mfranzke — APPROVED** [review]
